### PR TITLE
Edit the contributing docs

### LINKED ISSUE #1047: Bug: inserting fixtures doesn't work anymore
**Describe the bug**

In the documentation, there's a section about [loading fixtures](https://docs.joinmbin.org/contributing/#fixtures). Running the command (`bin/console doctrine:fixtures:load --append --no-debug`) however doesn't work.

**On which Mbin instance did you find the bug?**

In dev environment

**Which Mbin version was running on the instance?**

main @  https://github.com/MbinOrg/mbin/commit/a103bd7cff31c93953e741ee2f1e8e970400ca93

**To Reproduce**

1. Start the dev environment using docker
2. run `docker compose exec php bin/console doctrine:fixtures:load --append --no-debug`
3. look at the logs in `messenger` using `docker compose logs --tail 100 --follow messenger` in another console
4. see errors spamming
5. stop the `bin/console doctrine:fixtures:load` command

**Expected behavior**

Not sure...

**Screenshots**

N/A

**Additional context**

<details><summary>Errors</summary>
<p>

```
messenger.WARNING: Error thrown while handling message App\Message\ActivityPub\Outbox\CreateMessage. Sending for retry #2 using 1200000 ms delay. Error: "Handling "App\Message\ActivityPub\Outbox\CreateMessage" failed: App\Service\ActivityPub\Wrapper\CreateWrapper::build(): Argument #1 ($item) must be of type App\Entity\Contracts\ActivityPubActivityInterface, null given, called in /var/www/mbin/src/MessageHandler/ActivityPub/Outbox/CreateHandler.php on line 55" {"class":"App\\Message\\ActivityPub\\Outbox\\CreateMessage","message_id":null,"retryCount":2,"delay":1200000,"error":"Handling \"App\\Message\\ActivityPub\\Outbox\\CreateMessage\" failed: App\\Service\\ActivityPub\\Wrapper\\CreateWrapper::build(): Argument #1 ($item) must be of type App\\Entity\\Contracts\\ActivityPubActivityInterface, null given, called in /var/www/mbin/src/MessageHandler/ActivityPub/Outbox/CreateHandler.php on line 55","exception":"[object] (Symfony\\Component\\Messenger\\Exception\\HandlerFailedException(code: 0): Handling \"App\\Message\\ActivityPub\\Outbox\\CreateMessage\" failed: App\\Service\\ActivityPub\\Wrapper\\CreateWrapper::build(): Argument #1 ($item) must be of type App\\Entity\\Contracts\\ActivityPubActivityInterface, null given, called in /var/www/mbin/src/MessageHandler/ActivityPub/Outbox/CreateHandler.php on line 55 at /var/www/mbin/vendor/symfony/messenger/Middleware/HandleMessageMiddleware.php:124)\n[previous exception] [object] (TypeError(code: 0): App\\Service\\ActivityPub\\Wrapper\\CreateWrapper::build(): Argument #1 ($item) must be of type App\\Entity\\Contracts\\ActivityPubActivityInterface, null given, called in /var/www/mbin/src/MessageHandler/ActivityPub/Outbox/CreateHandler.php on line 55 at /var/www/mbin/src/Service/ActivityPub/Wrapper/CreateWrapper.php:31)"} []
07:39:59 WARNING   [messenger] Error thrown while handling message App\Message\ActivityPub\Outbox\CreateMessage. Sending for retry #2 using 1200000 ms delay. Error: "Handling "App\Message\ActivityPub\Outbox\CreateMessage" failed: App\Service\ActivityPub\Wrapper\CreateWrapper::build(): Argument #1 ($item) must be of type App\Entity\Contracts\ActivityPubActivityInterface, null given, called in /var/www/mbin/src/MessageHandler/ActivityPub/Outbox/CreateHandler.php on line 55"
[
  "class" => "App\Message\ActivityPub\Outbox\CreateMessage",
  "message_id" => null,
  "retryCount" => 2,
  "delay" => 1200000,
  "error" => "Handling "App\Message\ActivityPub\Outbox\CreateMessage" failed: App\Service\ActivityPub\Wrapper\CreateWrapper::build(): Argument #1 ($item) must be of type App\Entity\Contracts\ActivityPubActivityInterface, null given, called in /var/www/mbin/src/MessageHandler/ActivityPub/Outbox/CreateHandler.php on line 55",
  "exception" => Symfony\Component\Messenger\Exception\HandlerFailedException {
    #message: "Handling "App\Message\ActivityPub\Outbox\CreateMessage" failed: App\Service\ActivityPub\Wrapper\CreateWrapper::build(): Argument #1 ($item) must be of type App\Entity\Contracts\ActivityPubActivityInterface, null given, called in /var/www/mbin/src/MessageHandler/ActivityPub/Outbox/CreateHandler.php on line 55"
    #code: 0
    #file: "./vendor/symfony/messenger/Middleware/HandleMessageMiddleware.php"
    #line: 124
    -previous: TypeError {#1
      #message: "App\Service\ActivityPub\Wrapper\CreateWrapper::build(): Argument #1 ($item) must be of type App\Entity\Contracts\ActivityPubActivityInterface, null given, called in /var/www/mbin/src/MessageHandler/ActivityPub/Outbox/CreateHandler.php on line 55"
      #code: 0
      #file: "./src/Service/ActivityPub/Wrapper/CreateWrapper.php"
      #line: 31
      trace: {
        ./src/Service/ActivityPub/Wrapper/CreateWrapper.php:31 {
          App\Service\ActivityPub\Wrapper\CreateWrapper->build(ActivityPubActivityInterface $item): array
          › ])]
          › public function build(ActivityPubActivityInterface $item): array
          › {
        }
        ./src/MessageHandler/ActivityPub/Outbox/CreateHandler.php:55 { …}
        ./src/MessageHandler/MbinMessageHandler.php:29 { …}
        ./vendor/doctrine/dbal/src/Connection.php:1282 { …}
        ./src/MessageHandler/MbinMessageHandler.php:29 { …}
        ./src/MessageHandler/ActivityPub/Outbox/CreateHandler.php:44 { …}
        ./vendor/symfony/messenger/Middleware/HandleMessageMiddleware.php:152 { …}
        ./vendor/symfony/messenger/Middleware/HandleMessageMiddleware.php:91 { …}
        ./vendor/symfony/messenger/Middleware/SendMessageMiddleware.php:71 { …}
        ./vendor/symfony/messenger/Middleware/FailedMessageProcessingMiddleware.php:34 { …}
        ./vendor/symfony/messenger/Middleware/DispatchAfterCurrentBusMiddleware.php:68 { …}
        ./vendor/symfony/messenger/Middleware/RejectRedeliveredMessageMiddleware.php:41 { …}
        ./vendor/symfony/messenger/Middleware/AddBusNameStampMiddleware.php:35 { …}
        ./vendor/symfony/messenger/Middleware/TraceableMiddleware.php:36 { …}
        ./vendor/symfony/messenger/MessageBus.php:70 { …}
        ./vendor/symfony/messenger/TraceableMessageBus.php:38 { …}
        ./vendor/symfony/messenger/RoutableMessageBus.php:54 { …}
        ./vendor/symfony/messenger/Worker.php:162 { …}
        ./vendor/symfony/messenger/Worker.php:109 { …}
        ./vendor/symfony/messenger/Command/ConsumeMessagesCommand.php:244 { …}
        ./vendor/symfony/console/Command/Command.php:279 { …}
        ./vendor/symfony/console/Application.php:1047 { …}
        ./vendor/symfony/framework-bundle/Console/Application.php:123 { …}
        ./vendor/symfony/console/Application.php:316 { …}
        ./vendor/symfony/framework-bundle/Console/Application.php:77 { …}
        ./vendor/symfony/console/Application.php:167 { …}
        ./vendor/symfony/runtime/Runner/Symfony/ConsoleApplicationRunner.php:49 { …}
        ./vendor/autoload_runtime.php:29 { …}
        ./bin/console:15 { …}
      }
    }
    -envelope: Symfony\Component\Messenger\Envelope {#2
      -stamps: [
        "Symfony\Component\Messenger\Stamp\BusNameStamp" => [
          Symfony\Component\Messenger\Stamp\BusNameStamp {
            -busName: "messenger.bus.default"
          }
        ],
        "Symfony\Component\Messenger\Stamp\DelayStamp" => [
          Symfony\Component\Messenger\Stamp\DelayStamp {
            -delay: 300000
          }
        ],
        "Symfony\Component\Messenger\Stamp\ErrorDetailsStamp" => [
          Symfony\Component\Messenger\Stamp\ErrorDetailsStamp {
            -exceptionClass: "TypeError"
            -exceptionCode: 0
            -exceptionMessage: "App\Service\ActivityPub\Wrapper\CreateWrapper::build(): Argument #1 ($item) must be of type App\Entity\Contracts\ActivityPubActivityInterface, null given, called in /var/www/mbin/src/MessageHandler/ActivityPub/Outbox/CreateHandler.php on line 55"
            -flattenException: null
          }
        ],
        "Symfony\Component\Messenger\Stamp\RedeliveryStamp" => [
          Symfony\Component\Messenger\Stamp\RedeliveryStamp {
            -redeliveredAt: DateTimeImmutable @1724139299 {
              date: 2024-08-20 07:34:59.0 +00:00
            }
            -retryCount: 1
          }
        ],
        "Symfony\Component\Messenger\Stamp\SerializedMessageStamp" => [
          Symfony\Component\Messenger\Stamp\SerializedMessageStamp {
            -serializedMessage: "{"id":393,"type":"App\\Entity\\PostComment"}"
          }
        ],
        "Symfony\Component\Messenger\Bridge\Amqp\Transport\AmqpReceivedStamp" => [
          Symfony\Component\Messenger\Bridge\Amqp\Transport\AmqpReceivedStamp {
            -amqpEnvelope: AMQPEnvelope {
              -contentType: "application/json"
              -contentEncoding: null
              -headers: [
                "X-Message-Stamp-Symfony\Component\Messenger\Stamp\BusNameStamp" => "[{"busName":"messenger.bus.default"}]",
                "X-Message-Stamp-Symfony\Component\Messenger\Stamp\DelayStamp" => "[{"delay":300000}]",
                "X-Message-Stamp-Symfony\Component\Messenger\Stamp\ErrorDetailsStamp" => "[{"exceptionClass":"TypeError","exceptionCode":0,"exceptionMessage":"App\\Service\\ActivityPub\\Wrapper\\CreateWrapper::build(): Argument #1 ($item) must be of type App\\Entity\\Contracts\\ActivityPubActivityInterface, null given, called in \/var\/www\/mbin\/src\/MessageHandler\/ActivityPub\/Outbox\/CreateHandler.php on line 55","flattenException":null}]",
                "X-Message-Stamp-Symfony\Component\Messenger\Stamp\RedeliveryStamp" => "[{"retryCount":1,"redeliveredAt":"2024-08-20T07:34:59+00:00"}]",
                "type" => "App\Message\ActivityPub\Outbox\CreateMessage",
                "x-death" => [
                  [
                    "count" => 1,
                    "reason" => "expired",
                    "queue" => "delay_outbox_outbox_300000_retry",
                    "time" => AMQPTimestamp {
                      -timestamp: 1724139599.0
                    },
                    "exchange" => "delays",
                    "routing-keys" => [
                      "delay_outbox_outbox_300000_retry"
                    ]
                  ]
                ],
                "x-first-death-exchange" => "delays",
                "x-first-death-queue" => "delay_outbox_outbox_300000_retry",
                "x-first-death-reason" => "expired",
                "x-last-death-exchange" => "delays",
                "x-last-death-queue" => "delay_outbox_outbox_300000_retry",
                "x-last-death-reason" => "expired"
              ]
              -deliveryMode: 2
              -priority: 0
              -correlationId: null
              -replyTo: null
              -expiration: null
              -messageId: null
              -timestamp: 1724139299
              -type: null
              -userId: null
              -appId: null
              -clusterId: null
              -body: "{"id":393,"type":"App\\Entity\\PostComment"}"
              -consumerTag: ""
              -deliveryTag: 1504
              -isRedelivery: false
              -exchangeName: ""
              -routingKey: "outbox"
              -delivery_mode: 2 (persistent)
            }
            -queueName: "outbox"
          }
        ],
        "Symfony\Component\Messenger\Stamp\ReceivedStamp" => [
          Symfony\Component\Messenger\Stamp\ReceivedStamp {
            -transportName: "outbox"
          }
        ],
        "Symfony\Component\Messenger\Stamp\ConsumedByWorkerStamp" => [
          Symfony\Component\Messenger\Stamp\ConsumedByWorkerStamp {}
        ],
        "Symfony\Component\Messenger\Stamp\AckStamp" => [
          Symfony\Component\Messenger\Stamp\AckStamp {
            -ack: Closure(Envelope $envelope, ?Throwable $e = null) {
              class: "Symfony\Component\Messenger\Worker"
              this: Symfony\Component\Messenger\Worker { …}
              use: {
                $transportName: "outbox"
                $acked: & false
              }
              file: "./vendor/symfony/messenger/Worker.php"
              line: "155 to 158"
            }
          }
        ]
      ]
      -message: App\Message\ActivityPub\Outbox\CreateMessage {#4
        +id: 393
        +type: "App\Entity\PostComment"
      }
    }
    -exceptions: [
      "App\MessageHandler\ActivityPub\Outbox\CreateHandler::__invoke" => TypeError {#1}
    ]
    trace: {
      ./vendor/symfony/messenger/Middleware/HandleMessageMiddleware.php:124 { …}
      ./vendor/symfony/messenger/Middleware/SendMessageMiddleware.php:71 { …}
      ./vendor/symfony/messenger/Middleware/FailedMessageProcessingMiddleware.php:34 { …}
      ./vendor/symfony/messenger/Middleware/DispatchAfterCurrentBusMiddleware.php:68 { …}
      ./vendor/symfony/messenger/Middleware/RejectRedeliveredMessageMiddleware.php:41 { …}
      ./vendor/symfony/messenger/Middleware/AddBusNameStampMiddleware.php:35 { …}
      ./vendor/symfony/messenger/Middleware/TraceableMiddleware.php:36 { …}
      ./vendor/symfony/messenger/MessageBus.php:70 { …}
      ./vendor/symfony/messenger/TraceableMessageBus.php:38 { …}
      ./vendor/symfony/messenger/RoutableMessageBus.php:54 { …}
      ./vendor/symfony/messenger/Worker.php:162 { …}
      ./vendor/symfony/messenger/Worker.php:109 { …}
      ./vendor/symfony/messenger/Command/ConsumeMessagesCommand.php:244 { …}
      ./vendor/symfony/console/Command/Command.php:279 { …}
      ./vendor/symfony/console/Application.php:1047 { …}
      ./vendor/symfony/framework-bundle/Console/Application.php:123 { …}
      ./vendor/symfony/console/Application.php:316 { …}
      ./vendor/symfony/framework-bundle/Console/Application.php:77 { …}
      ./vendor/symfony/console/Application.php:167 { …}
      ./vendor/symfony/runtime/Runner/Symfony/ConsoleApplicationRunner.php:49 { …}
      ./vendor/autoload_runtime.php:29 { …}
      ./bin/console:15 {
        › 
        › require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
        › 
        arguments: {
          "/var/www/mbin/vendor/autoload_runtime.php"
        }
      }
    }
  }
]
```

</p>
</details>

### PR DESCRIPTION
**Pull request by BentiGorlich**
- add info to stop the messengers

Closes #1047

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -85,7 +85,10 @@ SYMFONY_DEPRECATIONS_HELPER=disabled ./bin/phpunit tests/Unit
 ### Fixtures
 
 You might want to load random data to database instead of manually adding magazines, users, posts, comments etc.
-To do so, execute: `bin/console doctrine:fixtures:load --append --no-debug`
+To do so, execute: `php bin/console doctrine:fixtures:load --append --no-debug`
+You should stop the messenger processes, as they are not able to access the data while it is being created, but try to do so.
+- Docker: `docker compose down messenger`
```
Comment (LoveIsGrief) [inline CONTRIBUTING.md:90]:
```suggestion
- Docker: `docker compose stop messenger`
```

No need to completely take it down

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -85,7 +85,10 @@ SYMFONY_DEPRECATIONS_HELPER=disabled ./bin/phpunit tests/Unit
 ### Fixtures
 
 You might want to load random data to database instead of manually adding magazines, users, posts, comments etc.
-To do so, execute: `bin/console doctrine:fixtures:load --append --no-debug`
+To do so, execute: `php bin/console doctrine:fixtures:load --append --no-debug`
+You should stop the messenger processes, as they are not able to access the data while it is being created, but try to do so.
```
Comment (LoveIsGrief) [inline CONTRIBUTING.md:89]: Just for my understanding: why aren't they able to access the data?

Comment (BentiGorlich) [reply CONTRIBUTING.md:89]: I added that explanation to the issue

Comment (LoveIsGrief) [reply CONTRIBUTING.md:89]: Thanks, that's clear.

**Review by LoveIsGrief — APPROVED** [review]
LGTM!

**Review by melroy89 — APPROVED** [review]

**Pull request merged**